### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by tkhattar** [LLM360/k2-train#2]
Could we add a requirements.txt file? Right now it is unclear what is needed to run the script. 

The same applies to the data prep repo.

**Comment by hunterhector** [comment]
It would be helpful for us to point out the docker file too? @hwang595, @yukiontheiceberg do we have a public host of our docker image?

**Comment by hwang595** [comment]
Yes @hunterhector, here you go: `https://hub.docker.com/repository/docker/zazzyyyy/project-k2/general`. And please use `tag 0.2`.
